packages/search-bar/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@vtex/search-bar",
+  "version": "0.1.0",
+  "description": "Search bar of the store",
+  "main": "dist/index.js",
+  "files": [
+    "dist/"
+  ],
+  "peerDependencies": {
+    "prop-types": "^15.6.1",
+    "react": "^16.3.2",
+    "react-dom": "^16.3.2"
+  },
+  "dependencies": {
+    "@vtex/styleguide": "2.0.0-rc.34",
```

Comment:
Vtex styleguide can already be used importing directly inside de code, without npm dependencies.